jena-arq/src/main/java/org/apache/jena/atlas/web/TypedInputStream.java

```diff
@@ -31,6 +31,7 @@
     // May be different from the URI used to access the resource 
     // e.g. 303 redirection, mapped URI redirection 
     private String baseURI ;
+    private boolean isClosed = false ;
```

Comment:
Ok, cool. I tend to be liberal with it, unless I am worried about a specific performance problem.